pandas/tests/groupby/test_groupby_shift_diff.py

```diff
@@ -113,3 +113,33 @@ def test_group_diff_object_raises(object_dtype):
     )
     with pytest.raises(TypeError, match=r"unsupported operand type\(s\) for -"):
         df.groupby("a")["b"].diff()
+        
+@pytest.mark.parametrize(
```

Comment:
since this is just a single test no need to parameterize (e.g. just create it inside the test)